### ISSUE TITLE
Press 1 or 2 to select weapon

### ISSUE THREAD

**Issue by Russianstory** [UnofficialCrusaderPatch/UnofficialCrusaderPatch#1026]
![нажать 1 или 2 для выбора оружия](https://github.com/UnofficialCrusaderPatch/UnofficialCrusaderPatch/assets/48765277/0ab8b364-72b5-4c4d-9c25-f8db7bd1b18d)

**Comment by gynt** [comment]
What are you trying to say?

**Comment by GRhin** [comment]
I'll let someone who is more active in the team give a definitive answer (I
haven't been active for a long time), but one feature that is high on the
wish list is a customisable hotkey feature. This would include an option
for this, and your other recent issues.
I have no idea on where the devs are on that feature, or if it's been
forgotten. I do know that shcplayer mod has this already, and ucp is
currently in the process of converting the code to work along the same
lines that shcplayer does, so it is theoretically possible.

**Comment by Russianstory** [comment]
Yes, I myself am not so often here now, but thanks for taking these moments into account. I hope the community will appreciate these innovations

**Comment by gynt** [comment]
Definitely feasible with the upcoming UCP stuff. We are indeed working on a general approach that includes keybindings
